mayastor/src/subsys/nvmf/admin_cmd.rs

```diff
@@ -49,7 +49,11 @@ extern "C" fn nvmf_create_snapshot_hdlr(req: *mut spdk_nvmf_request) -> i32 {
                 | (cmd.__bindgen_anon_2.cdw11 as u64) << 32
         };
         let snapshot_name = format!("{}-snap-{}", bd.name(), snapshot_time);
-        replica.create_snapshot(req, &snapshot_name);
+        // Blobfs operations must be on md_thread
+        let mgmt_reactor = Reactors::get_by_core(Cores::first()).unwrap();
```

Comment:
btw, you can just use `Reactors::master().send_future`

Reply:
Thanks, fixed.